java/lib/com/google/protobuf/protobuf-java/maven-metadata-local.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<metadata>
+  <groupId>com.google.protobuf</groupId>
+  <artifactId>protobuf-java</artifactId>
+  <versioning>
+    <release>2.6.1</release>
+    <versions>
+      <version>2.6.1</version>
+    </versions>
+    <lastUpdated>20151102180249</lastUpdated>
```

Comment:
As discussed, this is used by maven to check against, e.g., ~/.m2/repository/com/google/protobuf/protobuf-java/2.6.1/protobuf-java-2.6.1.jar.lastUpdated - to determine whether a local version is newer or older than the one checked into our repo.